package.json

```diff
@@ -32,6 +32,7 @@
     "tailwindcss": "^1.6.2"
   },
   "devDependencies": {
+    "@fullhuman/postcss-purgecss": "^2.3.0",
```

Comment:
I don't think we can use this here as we won't be using all of our generated css.

Reply:
We can use this in our projects that use this version of decanter but ootb we will need to have everything outputted.

Reply:
Yup - agree. Just testing. I've removed the purge stuff and saving the code for production use 😺